Build/BuildScripts/CodeJam.AppVeyor.NUnit.Tests.ps1

```diff
@@ -1,22 +1,21 @@
 ﻿$include = "*.Tests.dll", "*.Tests.NUnit.dll"
-$includePerfTests = "*.Tests.Performance.dll"
-$exclude = "Experimental\\.*?\\CodeJam.Tests.Performance.dll"
 
 mkdir "$env:APPVEYOR_BUILD_FOLDER\_Results" -ErrorAction SilentlyContinue
 
 $wc = New-Object System.Net.WebClient
 
-$targetsDotNet = "net47","net46","net45","net40","net35","net20"
+#run .net tests
+$targetsDotNet = "net472","net461","net45","net40","net35","net20"
```

Comment:
Should we add net48, net471, net470 too ?

Reply:
Done.